Formula/pulledpork.rb

```diff
@@ -1,10 +1,9 @@
 class Pulledpork < Formula
   desc "Snort rule management"
   homepage "https://github.com/shirkdog/pulledpork"
-  url "https://github.com/shirkdog/pulledpork/archive/v0.7.3.tar.gz"
-  sha256 "48c66dc9abb7545186d4fba497263c1d1b247c0ea7f0953db4d515e7898461a2"
-  license "GPL-2.0"
-  revision 3
+  url "https://github.com/shirkdog/pulledpork/archive/v0.7.4.tar.gz"
+  sha256 "f0149eb6f723b622024295e0ee00e1acade93fae464b9fdc323fdf15e99c388c"
+  license "GPL-2.0-or-later"
```

Comment:
Seems legit. I double-checked the files under `contrib` and those look compatible, too.